qemu/tests/fio_perf.py

```diff
@@ -127,6 +148,9 @@ def _pin_vm_threads(vm, node):
     _pin_vm_threads(vm, params.get("numa_node"))
 
     # get parameter from dictionary
+    check_install_fio = params["check_install_fio"]
+    tarball = params["tarball"]
+    compile_cmd = params.get("compile_cmd")
```

Comment:
Why not get these there params directly in fio_install()? You should be able to use all parameters in run() function globally, they are in the same context.